INAD_API_extraction.yaml

```diff
@@ -16,14 +17,24 @@ info:
 
     - [NONBLOCK_PULL_REST] Not Blocking Pull REST
   contact:
+  
+    # da verificare  
     name: Infocamere S.C.p.A.
     url: https://www.infocamere.it/
   license:
+  
+    # da verificare    
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0
+    
+  # eliminare -SNAPSHOT im contrasto con [RAC_GEN_002] Endpoint delle API, vedi documento operativo “Raccomandazioni di implementazione” in https://www.agid.gov.it/sites/default/files/repository_files/linee_guida_interoperabilit_tecnica_pa.pdf
   version: 1.0.0-SNAPSHOT
 servers:
+
+  # da consolidare considerando che il prefisso /rest/inad/v1/domiciliodigitale è da includere nell'URL 
   - url: https://domiciliodigitaleapi.oscl.infocamere.it
+  
+  # da verificare
```

Comment:
TODO

Reply:
L'url definitiva sarà quella di produzione: https://api.inad.gov.it (da definire)

L'attuale url è quella di collaudo. Per completare la prima versione delle specifiche consigliamo di modificare l'URL con quella sopra riportata.
A regime sarà quella dell'ambiente di produzione https://api.inad.gov.it.